pkg/cache/clusterqueue_test.go

```diff
@@ -1074,30 +1075,38 @@ func TestDominantResourceShare(t *testing.T) {
 }
 
 func TestCohortLendable(t *testing.T) {
-	cq := clusterQueue{
-		Cohort: &cohort{
-			Members: sets.New(
-				&clusterQueue{
-					Lendable: map[corev1.ResourceName]int64{
-						corev1.ResourceCPU: 8_000,
-						"example.com/gpu":  3,
-					},
-				},
-				&clusterQueue{
-					Lendable: map[corev1.ResourceName]int64{
-						corev1.ResourceCPU: 2_000,
-					},
-				},
-			),
-		},
+	cache := New(utiltesting.NewFakeClient())
```

Comment:
Why is the test changed? If the previous test case still passes maybe we could have 2 test cases (old and new) to make it clear there is no regression in coverage.

Reply:
deleted the underlying field - it was relying on ClusterQueue.lendable

Reply:
ah got